nox/virtualenv.py

```diff
@@ -168,7 +168,7 @@ class PassthroughEnv(ProcessEnv):
     @staticmethod
     def is_offline() -> bool:
         """As of now this is only used in conda_install"""
-        return CondaEnv.is_offline()  # pragma: no cover
+        return CondaEnv.is_offline()
```

Comment:
This is the one remaining uncovered line. Maybe just revert this change, as it's not really relevant to the susbtance of this PR.